single_select_plugin/src/dp_single_select_plugin/Plugin.tsx

```diff
@@ -0,0 +1,45 @@
+// (C) 2021-2022 GoodData Corporation
+import { DashboardContext, DashboardPluginV1, IDashboardCustomizer } from "@gooddata/sdk-ui-dashboard";
+import { PluginAttributeFilter } from "./components/PluginAttributeFilter";
+
+import entryPoint from "../dp_single_select_plugin_entry";
+
+export class Plugin extends DashboardPluginV1 {
+    public readonly author = entryPoint.author;
+    public readonly displayName = entryPoint.displayName;
+    public readonly version = entryPoint.version;
+    public readonly minEngineVersion = entryPoint.minEngineVersion;
+    public readonly maxEngineVersion = entryPoint.maxEngineVersion;
+
+    public onPluginLoaded(_ctx: DashboardContext, _parameters?: string): Promise<void> | void {
```

Comment:
I would remove the methods that are not used. I.e. `onPluginLoaded` and `onPluginUnload`.